src/renderer/components/+workloads-pods/pod-tolerations.tsx

```diff
@@ -66,20 +59,24 @@ export function PodTolerations({ tolerations }: Props) {
     <Table
       tableId="workloads_pod_tolerations"
       selectable
+      items={tolerations}
```

Comment:
does this change behaviour?

Reply:
No, it is specifically designed not to. This is just a better way (from a typescript perspective) for type inference.